config/default.js

```diff
@@ -124,6 +124,11 @@ module.exports = {
       save: 'save',
       update: 'update',
     },
+    setup: {
+      path: '/create-admin',
```

Comment:
I agree, however I believe we need some settings which maybe aren't able to be changed by an implementor, but that are still settings. For instance, this path (`/create-admin`) is in a bunch of places in four different files:

```
lib/init/authenticated.js
17:  ... req.url !== config.authentication.login.path ...

lib/routes/login.js
32:          return res.redirect(config.users.setup.path);

lib/routes/users.js
28:    app.get(config.users.setup.path, users.routes.setup);
33:    app.post(config.users.setup.path, multipartMiddleware, users.routes.first);
38:    app.get(`/${config.users.setup.path}/${config.users.actions.save}`, users.routes.setup);

lib/users/routes.js
43:          action: config.users.setup.path,
233:  if (_.includes(referrer, config.users.setup.path)) {
```

It feels wrong to simply hard-code something like a URL path throughout the system. Because if we ever do want to change it, we're on a hunt to find it in all the different places. Same could be said for `config.authentication.login.path` or any number of other system settings that we don't _really_ want to change, but also don't want to keep track of the places they exist in case we ever do.

Regardless, to get this PR complete, I will make this change. But, I think we should consider some internal conglomeration of settings for things like this.